andela/merry-ah-frontend
# #164215435 implement feedback from TTL and Product owner

#### What does this PR do?
Implement Feedback from Product owner and TTL

#### Description of Task to be completed?
- refactor query string function
- refactor tests for query string function
- add new logo
- remove spaces from featured arts slider
- fix slider to be continuous without breaking
- add new tests for the slider
- add social login to the signup page

#### How should this be manually tested?
- Run `npm i`
- Start the server `npm run start:dev`
- Navigate to `localhost:5050` to see changes

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#164215435](https://www.pivotaltracker.com/story/show/164215435)
#### Screenshots (if appropriate)
<img width="1412" alt="screen shot 2019-02-27 at 8 33 50 pm" src="https://user-images.githubusercontent.com/7751635/53517510-136c0b80-3acf-11e9-95a5-476f88d8d48e.png">
